docs/content/dagster-cloud/managing-deployments/branch-deployments.mdx

```diff
@@ -63,6 +63,84 @@ Branch Deployments aren't currently supported for use in Dagster Open Source.
 
 Output created from a branch deployment - such as a database, table, etc. - won't be automatically removed from storage once a branch is merged or closed. Refer to the [Best practices section](#best-practices) for info on how to handle this.
 
+### Change Tracking
+
+When a Branch Deployment is deployed, it compares the asset definitions in the branch deployment with the asset definitions in the main deployment. New assets or assets that have been changed in the branch are marked in the UI. The following sections cover what types of changes are detected.
+
+#### New assets
+
+Change Tracking will indicate if an asset is new in the Branch Deployment. <Image src="/images/dagster-cloud/managing-deployments/change-tracking-new.png" width={764} height={308} />
+
+#### Code version changes
+
+Change Tracking will indicate if the `code_version` of an asset has changed. Change Tracking can detect added, removed, or updated code versions. For example, if the main branch contains `customers` with code version `v1`
+
+```python file=/dagster_cloud/branch_deployments/change_tracking_code_version.py startafter=start_main_deployment endbefore=end_main_deployment dedent=4
+@asset(code_version="v1")
+def customers(): ...
+```
+
+And the pull request modifies `customers` and changes the code version to `v2`:
+
+```python file=/dagster_cloud/branch_deployments/change_tracking_code_version.py startafter=start_branch_deployment endbefore=end_branch_deployment dedent=4
+@asset(code_version="v2")
+def customers(): ...
+```
+
+The Branch Deployment will mark `customers` as changed. <Image src="/images/dagster-cloud/managing-deployments/change-tracking-code-version.png" width={1042} height={444} />
+
+Some dagster integrations, like `dagster-dbt`, automatically compute code versions for you. For more information on code versions, including strategies for how to compute code versions automatically, see the code versions documentation LINK.
+
+<!-- TODO - question about code versions - are they set to the latest run id if not user specified? docs claim yes but i only see code versions in the ui for dbt assets -->
+
+#### Dependency changes
+
+Change Tracking will indicate if the set of upstream dependencies for an asset has changed. Change Tracking can detect added or removed upstream dependencies. For example, if the main branch contains `returns`:
```

Comment:
interested in feedback on the description of this type of change. This is definitely the most nuanced type of change we detect so i want to make sure the description is clear. let me know if you find it confusing and i can then start clarifying it accordingly